src/integration-tests/src/sql.rs

```diff
@@ -113,22 +114,26 @@ async fn cleanup_stale_sql_instances(
         .set_filter(format!(
             "name:{PREFIX}* AND settings.userLabels.{INSTANCE_LABEL}:true"
         ))
-        .send()
-        .await?;
+        .by_item()
+        .into_stream();
 
     let pending_deletion = instances
-        .items
-        .into_iter()
-        .filter_map(|instance| {
-            if instance.create_time? < stale_deadline {
-                Some(client.delete(project_id, instance.name).send())
-            } else {
-                None
+        .filter_map(|instance| async {
+            match instance {
+                Ok(instance) => {
+                    if instance.create_time?.le(&stale_deadline) {
```

Comment:
Using  `.le()` does not look very readable to me. Can we use `instance.create_time? < state_deadline`?